### PR TITLE
Fix dashboard empty content

### PR DESCRIPTION
**Pull request by GretaD**
maildashboard
dashboar1
dash2

### CODE REVIEW

File: src/views/Dashboard.vue [hunk 1]
```diff
@@ -112,19 +113,8 @@ export default {
 }
 </script>
 
-<style lang="scss" scoped>
-.empty-content {
-	text-align: center;
+<style lang="scss">
+#mail--empty-content {
 	margin-top: 50px;
```
Comment (juliushaertl) [inline src/views/Dashboard.vue:118]:
```suggestion
	text-align: center;
	margin-top: 5vh;
```

File: src/views/Dashboard.vue [hunk 2]
```diff
@@ -38,13 +38,14 @@
...
-				<p class="empty-content__text">
-					{{ t('mail', 'No messages found yet') }}
-				</p>
-				<a v-if="accounts.length === 0" :href="accountSetupUrl" class="button">{{ t('mail', 'Set up an account') }}</a>
-			</div>
+			<EmptyContent id="mail--empty-content" icon="icon-checkmark">
+				<template #desc>
+					{{ t('mail', 'No message found yet') }}
+				</template>
+		</EmptyContent>
```
Comment (juliushaertl) [inline src/views/Dashboard.vue:45]: Indentation is off

File: src/views/Dashboard.vue [hunk 3]
```diff
@@ -38,13 +38,14 @@
...
+<div class="no-account icon-archive">
+			<a v-if="accounts.length === 0" :href="accountSetupUrl" class="button">{{ t('mail', 'Set up an account') }}</a>
```
Comment (juliushaertl) [inline src/views/Dashboard.vue:47]: This should be part of the empty content view as well

Comment (juliushaertl) [reply src/views/Dashboard.vue:47]: It will also then center the button properly

Comment (GretaD) [reply src/views/Dashboard.vue:47]: hmm, i cannot style it while inside, thats why i removed it, but let me try again

Comment (GretaD) [reply src/views/Dashboard.vue]: it works with div :)

File: src/views/Dashboard.vue [hunk 4]
```diff
@@ -55,6 +56,7 @@ import { generateUrl, imagePath } from '@nextcloud/router'
 import { DashboardWidget, DashboardWidgetItem } from '@nextcloud/vue-dashboard'
 import orderBy from 'lodash/fp/orderBy'
 import prop from 'lodash/fp/prop'
+import EmptyContent from '@nextcloud/vue/dist/components/EmptyContent'
```
Comment (GretaD) [inline src/views/Dashboard.vue:59]: it says the same also for import EmptyContent from '@nextcloud/vue/dist/Components/EmptyContent'. Whats the right path?

Comment (juliushaertl) [reply src/views/Dashboard.vue:59]: Ah mail is still on version 1.5.0 but EmptyContent is only part of 2.0.0+

Comment (ChristophWurst) [reply src/views/Dashboard.vue:59]: Then we need https://github.com/nextcloud/mail/pull/3466

Comment (GretaD) [reply src/views/Dashboard.vue]: done

**Review by ChristophWurst — APPROVED** [review]
Looks good :)

Please squash

**Review by juliushaertl — APPROVED** [review]
